widgets/linechart/axes/label.go

```diff
@@ -89,7 +89,7 @@ func rowLabel(scale *YScale, y int, labelWidth int) (*Label, error) {
 	}
 
 	ar := rowLabelArea(y, labelWidth)
-	pos, err := align.Text(ar, v.Text(), align.HorizontalRight, align.VerticalMiddle)
+	pos, _ := align.Text(ar, v.Text(), align.HorizontalRight, align.VerticalMiddle)
```

Comment:
Thanks for fixing this, this was actually a bug. We however shouldn't silently discard the error variable as that can create a hard to debug scenario. I.e. the align.Text() function fails and reports an error, we ignore it and silently report success. The failure of align.Text() will most likely result in some unexpected behavior that will be hard to find.

If we correctly process and report the error, it will lead us to this very line when something goes wrong.

We can do something like this, adding additional context to the reported error:

pos, err := align.Text(ar, v.Text(), align.HorizontalRight, align.VerticalMiddle)
if err != nil {
    return nil, fmt.Errorf("unable to align the label value: %v", err)
}

Reply:
Hehe, I actually fixed that once, then I decided to revert it again... It is fixed now too. Glad we found a bug here!